agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -98,6 +98,17 @@ func TestOOMContainer(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+// TestOOMTask verifies that a task with a memory limit returns an error
```

Comment:
Can you add the `agent.RequireVersion(">=1.16.0")` for this test?

Reply:
fixed.